package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Seneca RabbitMQ transport",
   "main": "rabbitmq-transport.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha test/*.test.js"
+    "test": "./node_modules/.bin/lab test/*test.js  -r console -v",
```

Comment:
You don't need the node modules part, just lab. Also can you add the linting flag?